internal/pkg/cli/app_delete_test.go

```diff
@@ -85,3 +88,115 @@ func TestConfirmDelete(t *testing.T) {
 		})
 	}
 }
+
+func TestSourceInputs(t *testing.T) {
+	t.Run("should return errNoProjectInWorkspace", func(t *testing.T) {
+		opts := deleteAppOpts{
+			GlobalOpts: &GlobalOpts{
+				projectName: "",
```

Comment:
is this the expected output (i.e. empty string) when project does not exist?

Reply:
This is setting up the failure case, so it's expected input.